src/x25519.rs

```diff
@@ -187,6 +197,22 @@ pub const X25519_BASEPOINT_BYTES: [u8; 32] = [
     9, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
 ];
 
+/// This is a great helper struct for all our serde things. Hooray
+#[cfg_attr(feature = "serde", serde(crate = "our_serde"))]
+#[cfg_attr(
+    feature = "serde",
+    derive(our_serde::Serialize, our_serde::Deserialize)
+)]
+#[cfg_attr(feature = "serde", serde(remote = "Scalar"))]
+
+struct ScalarHelper(#[serde(getter = "Scalar::to_bytes")] [u8; 32]);
```

Comment:
Maybe this would be better named something like `AllowUnreducedScalarBytes` so that the name indicates the purpose really clearly, and the explanation in the FIXME comment above could be moved into its docstring, so that the docstring for the helper struct explains why it's required.

Reply:
Also, I think that the attribute will have to be
```
#[cfg_attr(feature = "serde", serde(getter = "Scalar::to_bytes"))]
```
to prevent a compilation failure when serde is disabled.